docs/modules/ROOT/pages/configuration-format.adoc

```diff
@@ -0,0 +1,94 @@
+= Configuration Format
+:description: The Hazelcast CLC recognizes the following items in the configuration file.
+
+{description}
+
+The configuration is in YAML format.
+It has the `cluster` and `ssl` sections.
+A typical configuration file looks as follows:
+
+```yaml
+cluster:
+  discovery-token: "XXXXXXXDOr9CWUNhzoyXXXXXXwK9nfRV8Ro4XUc6XXXXX"
+  name: "pr-abcd123"
+ssl:
+  ca-path: "ca.pem"
+  cert-path: "cert.pem"
+  key-path: "key.pem"
+  key-password: "766391XXXXX"
+```
+
+== cluster section
+
+[cols="1a,2a,1a"]
+|===
+|Key|Description|Default
+
+|address
+|Address of one of the members in the cluster.
+|`localhost:5701`
+
+|name
+|Cluster name.
+|`dev`
+
+|discovery-token
+|Viridian discovery token.
+|
+
+|api-base
+|Viridian API base URL or name.
+|
+
+|viridian-id
+|Viridian cluster ID.
+|
+
+|user
+|User name.
+|
+
+|password
+|Password.
+|
+
+|===
+
+== ssl section
+
+[cols="1a,2a,1a"]
+|===
+|Key|Description|Default
+
+|enabled
+|Set to `true` if SSL is enableed for this cluster. Automatically set to `true` if `cluster.discovery-token` is defined.
+|`false`
+
+|ca-path
+|CA path.
+|
+
+|cert-path
+|Certificate path.
+|
+
+|key-path
+|Key certificate path.
+|
+
+|key-password
+|Key certificate path.
+|
+
+|server
+|Server name if it's different from the certificate.
+|
+
+|skip-verify
+|Disables certificate verification. Shouldn't be used in production.
```

Comment:
Never use in a production environment.

Reply:
Updated at: https://github.com/hazelcast/hazelcast-commandline-client/pull/393/commits/7a4819939ba26d23c474a41583d98bedcc2ccba4